lib/YaST/workarounds.pm

```diff
@@ -77,4 +78,22 @@ sub apply_workaround_bsc1206132 {
     systemctl('daemon-reload');
 }
 
+sub workaround_poo124652_for_send_key_until {
```

Comment:
Please add head and description for the new function.

Reply:
done.